pkg/registry/clusterrolebinding/policybased/storage.go

```diff
@@ -0,0 +1,59 @@
+/*
```

Comment:
@kubernetes/sig-api-machinery does anyone want to comment on how to make a Strategy which accesses other registries in order to do Validation, and more specifically with this thread.